lib/go/test/epoch_test_helpers.go

```diff
@@ -712,6 +787,55 @@ func verifyEpochCommit(
 
 }
 
+// verifyEpochRecover verifies that the EpochRecover event values are equal to the provided expected values
+func verifyEpochRecover(
+	t *testing.T,
+	adapter *adapters.SDKAdapter,
+	epochAddress flow.Address,
+	expectedRecover EpochRecover) {
+	var emittedEvent EpochRecoverEvent
+	evtTypeStr := fmt.Sprintf("A.%s.FlowEpoch.EpochRecover", epochAddress.String())
```

Comment:
Use a `common.AddressLocation` and `common.Location.TypeID(...)` instead

Reply:
https://github.com/onflow/flow-core-contracts/pull/420/commits/5edced7a617bb1ecfd358e6e65d25f94acc3fdde